packages/modules/tax/src/migrations/Migration20240924114005.ts

```diff
@@ -0,0 +1,19 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240924114005 extends Migration {
+  async up(): Promise<void> {
+    this.addSql(
+      'alter table if exists "tax_rate" alter column "code" set not null;'
```

Comment:
if this wasn't required before and there is any row where `code is null`, this will fail.
If the value is always there, just disregard the comment.

Reply:
Nice catch, I'll include a backfilling strategy